sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2175,6 +2176,43 @@ pi_result piPluginGetLastError(char **message) {
   return pi2ur::piPluginGetLastError(message);
 }
 
+// Optimization strings.
+static const char *EmptyStr = "";
+static const char *NoOptStr = "-ze-opt-disable";
+static const char *O1OptStr = "-ze-opt-level=1";
+static const char *O2OptStr = "-ze-opt-level=2";
+
+// Returns plugin specific backend option.
+// Current support is only for optimization options.
+// Return '-ze-opt-disable' for frontend_option = -O0.
+// Return '-ze-opt-level=1' for frontend_option = -O1 or -O2.
+// Return '-ze-opt-level=2' for frontend_option = -O3.
+pi_result piPluginGetBackendOption(pi_platform platform,
+                                   const char *frontend_option,
+                                   const char **backend_option) {
+  using namespace std::literals;
+  if (frontend_option == nullptr) {
+    return PI_ERROR_INVALID_VALUE;
+  }
+  if (frontend_option == ""sv) {
+    *backend_option = EmptyStr;
+    return PI_SUCCESS;
+  }
+  if (frontend_option == "-O0"sv) {
+    *backend_option = NoOptStr;
```

Comment:
NIT: you don't actually need named static objects since string literals themselves have static lifetime
```suggestion
    *backend_option = "-ze-opt-disable";
```

Reply:
Would like to have aptly named variables for this. Thanks

Reply:
I'm +1'ing @smaslov-intel 's comment. I think direct string literal is much more readable.

Reply:
Addressed. Thanks